libs/wingc/src/type_check/lifts.rs

```diff
@@ -107,7 +107,10 @@ impl Lifts {
 			return;
 		}
 
-		let token = self.render_token(code);
+		let token = match lifted_thing {
+			Liftable::Expr(_) => self.render_token(code),
+			Liftable::Type(t) => self.render_token(&format!("{}", t)),
```

Comment:
Why do we now ignore the passed in `code` in this case?

Reply:
It resulted in unnecessarily verbose js where `$stdlib.core.toLiftableModuleType` is the code for the type.